models/zeroex/ethereum/zeroex_api_ethereum_schema.yml

```diff
@@ -0,0 +1,65 @@
+version: 2
+
+models:
+  - name: zeroex_api_ethereum_fills
+    meta:
+      blockchain: ethereum
+      project: zeroex
+      contributors: sui414, bakabhai993
+    config:
+      tags: ['ethereum','0x','dex_aggregator','dex','aggregator']
+    description: >
+        0x API erc20 swaps raw fills (including multihops) and also native swaps thru 0x exchange contracts (without using 0x API) on Ethereum
```

Comment:
You can add an unique keys testing here (after the `description`, and before the `columns`)
The test code looks like
```
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - tx_hash
            - evt_index
```